src/components/atoms/ModalFooter/ModalFooter.js

```diff
@@ -43,12 +43,12 @@ export default class ModalFooter extends HTMLElement {
 
     const btnExtraClasses = this.getAttribute('data-button-extra-classes');
 
-    const extraClasses = this.getAttribute('data-extra-classes');
+    const extraClasses = this.getAttribute('extra-classes');
```

Comment:
This is another example where we're changing how the attributes on the ModalFooter component work instead of focusing on the attributes on the button component. 

However, the attributes below (lines 49-51) are button component attributes with correct updates.